utils/utils.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import (
+	"regexp"
+	"time"
+	"strconv"
+  "log"
+  "github.com/microsoft/BladeMonRT/logging"
+)
+
+/** Class that contains utilities used in BladeMonRT classes. */
+type Utils struct {
+	logger *log.Logger
+}
+
+func NewUtils() *Utils {
+	var logger *log.Logger = logging.LoggerFactory{}.ConstructLogger("Utils")
+	return &Utils{logger : logger}
+}
+
+/** Class that represents an event parsed from event XML. */
+type EventFromXML struct {
```

Comment:
Rename to ETWEvent; put it in the WorkflowContext struct rather than duplicating the common fields

Reply:
Done